contracts/contracts/trees/AccQueue.sol

```diff
@@ -228,16 +231,12 @@ abstract contract AccQueue is Ownable, Hasher {
     subTreesMerged = false;
   }
 
-  /**
-   * A function that queues zeros to the specified level, hashes,
-   * the level, and enqueues the hash to the next level.
-   * @param _level The level at which to queue zeros.
-   */
+  /// @notice A function that queues zeros to the specified level, hashes,
+  /// the level, and enqueues the hash to the next level.
```

Comment:
what's the difference between the specified level and the level?

Reply:
it's the same. So we add the zero values to this level, hash it and then enqueue the hash to the next level